airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1654,6 +1659,11 @@ definitions:
         examples:
           - 0
           - 1
+      inject_on_first_request:
+        title: Inject Page Number
+        description: Using the `page number` with value `0` during the first request
```

Comment:
the value for the first request is not always `0` - it is defined by `start_from_page`